packages/htmlbars-syntax/lib/parser.js

```diff
@@ -4,12 +4,7 @@ import EntityParser from "../simple-html-tokenizer/entity-parser";
 import fullCharRefs from "../simple-html-tokenizer/char-refs/full";
 import nodeHandlers from "./node-handlers";
 import tokenHandlers from "./token-handlers";
-
-// this should be:
-// `import * from "../htmlbars-syntax";
-//
-// But this version of the transpiler does not support it properly
-module syntax from "../htmlbars-syntax";
+import * from "../htmlbars-syntax";
```

Comment:
is this line: https://github.com/tildeio/htmlbars/blob/bd68a56e126e58a4ba3f1254f7bb7f147e6740b8/packages/htmlbars-syntax/lib/parser.js#L36

tied to this change?

How should we go about implementing the same logic with the `import *` instead of `module syntax`?